packages/itwinui-variables/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@itwin/itwinui-variables",
+  "version": "0.1.0",
+  "author": "Bentley Systems",
+  "license": "MIT",
+  "main": "index.css",
+  "style": "index.css",
+  "files": [
+    "index.css",
```

Comment:
I think I asked that before, but we are sure we do not want to export our mixins, so users could include only parts of variables (if that makes sense) ?
I also assume, after this pckage is rleased, we will ask users to stop using itwinui-css and switch to this one? 
But then how variables inclusion would work (if user uses itwinui-react which takes itiwnui-css with variables and also uses itwinui-variables directly. Would it end up with duplicated declarations of variables? We need to test these combos in an app)